frontend/packages/console-shared/src/components/formik-fields/item-selector-field/ItemSelectorField.scss

```diff
@@ -5,6 +5,7 @@
     background: var(--pf-global--Color--light-200);
     padding: 4px;
     width: fit-content;
+    width: -moz-max-content;
```

Comment:
I think what you want is `display: inline-flex`
`display: flex` will be a block element therefore taking up the full available width, only to have use try to constrain the width by setting the `width` property. Sort of counter productive.
By using `display: inline-flex` will create an inline-block element which consumes only the space it needs